src/com/google/enterprise/gsafeed/ObjectFactory.java

```diff
@@ -0,0 +1,111 @@
+//
+// This file was generated by the JavaTM Architecture for XML Binding(JAXB) Reference Implementation, v2.2.8-b130911.1802 
+// See <a href="http://java.sun.com/xml/jaxb">http://java.sun.com/xml/jaxb</a> 
+// Any modifications to this file will be lost upon recompilation of the source schema. 
+// Generated on: 2017.09.25 at 11:57:15 AM PDT 
+//
+
+
+package com.google.enterprise.gsafeed;
+
+import javax.xml.bind.annotation.XmlRegistry;
+
+
+/**
+ * This object contains factory methods for each 
+ * Java content interface and Java element interface 
+ * generated in the com.google.enterprise.gsafeed package. 
+ * <p>An ObjectFactory allows you to programatically 
+ * construct new instances of the Java representation 
+ * for XML content. The Java representation of XML 
+ * content can consist of schema derived interfaces 
+ * and classes representing the binding of schema 
+ * type definitions, element declarations and model 
+ * groups.  Factory methods for each of these are 
+ * provided in this class.
+ * 
+ */
+@XmlRegistry
+public class ObjectFactory {
```

Comment:
Is this class going to be used?  For what purpose?  Why not just use "new Principal()" when wanting a Principal?

Reply:
It's auto-generated. I'm not seeing much use for it, either, but I think deleting it should wait.

Reply:
It apparently has/had in the past a use for containing boilerplate for constructing more-complicated objects defined with XML schemas.